packages/relay/src/lib/eth.ts

```diff
@@ -1698,6 +1698,11 @@ export class EthImpl implements Eth {
     } catch (e: any) {
       if (e instanceof SDKClientError && (e.isConnectionDropped() || e.isTimeoutExceeded())) {
         submittedTransactionId = e.transactionId || '';
+
+        // Log the target node account ID, right now, it's populated only for MaxAttemptsOrTimeout error
+        this.logger.info(
+          `${requestDetails.formattedRequestId} Transaction failed to execute against node with id: ${e.nodeAccountId}`
+        );
```

Comment:
This means we only log this when the errors are due to a timeout or a dropped connection. I know this is where the discussion originally began, but should we consider sending the error to `sendRawTransactionErrorHandler`? This way, we can log it there and expose the `nodeAccountId` along with all `SdkClient` errors.

Reply:
Done.